tasks/create_config.yml

```diff
@@ -18,15 +21,17 @@
   set_fact:
     listener_addr: "{{ lookup('vars', service_net_map[ (tripleo_role_name + 'MetricsQdrNetwork') | snake_case] + '_ip' ) }}"
 
-- name: Augment ssl_profiles with internal profile if required
-  set_fact:
-    ssl_profiles: "{{ ssl_profiles +
-                     [{ 'name': 'tlsProfile',
-                        'certFile': listener_ssl_cert_file,
-                        'keyFile': listener_ssl_key_file,
-                        'caCertFile': tripleo_internal_tlscafile
-                      }] }}"
+- name: Handle EnableInternalTLS
   when: tripleo_enable_internal_tls
+  block:
+    - name: Augment extra_ssl_profiles with internal profile
+      set_fact:
+        extra_ssl_profiles: "{{ extra_ssl_profiles +
```

Comment:
Not sure if it matters here, but did you consider using multi-line here so that you can start on the 2nd line and have your spacing even across the fact set? Might not be appropriate here, it just looked a little odd with the new lines and spacing.

```
extra_ssl_profiles: |
  "{{ extra_ssl_profiles +
  [{ ...

```

Reply:
I did not, but I'll give it a try and see how it looks

Reply:
These look janky no matter which way, but your way uses less horizontal space. patch incoming

Reply:
Oh, ansible, yaml, jinja, or whoever is responsible for this. I tried for a bit but am moving on to more productive things.

Tried
```
          connectors: |
            "{{ connectors +
            [{
                'host': item,
                'port': '5668',
                'role': 'edge',
                'verifyHostname': 'no',
                'saslMechanisms': 'ANONYMOUS'
            }] }}"
```
got `fatal: [qdrtest]: FAILED! => {"msg": "Unexpected templating type error occurred on (\"{{ connectors +\n[{\n    'host': item,\n    'port': '5668',\n    'role': 'edge',\n    'verifyHostname': 'no',\n    'saslMechanisms': 'ANONYMOUS'\n}] }}\"\n): must be str, not list"}`

Also tried
```
          connectors: >-
            {{ connectors +
            [{
                'host': item,
                'port': '5668',
                'role': 'edge',
                'verifyHostname': 'no',
                'saslMechanisms': 'ANONYMOUS'
            }] }}
```
and got `fatal: [qdrtest]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'str object' has no attribute 'items'"} ` (when it tries to iterate the list in the template)

Reply:
Ugh... that sucks :(  Sorry for the distraction!